pkg/eventfilter/subscriptionsapi/cesql_correlationid_filter_test.go

```diff
@@ -0,0 +1,235 @@
+/*
+Copyright 2022 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subscriptionsapi
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
```

Comment:
for unit tests and using the fake Kube client, you can import 

```
	kubeclient "knative.dev/pkg/client/injection/kube/client/fake"
```

and use it like it is the "normal" client, for example:

```
... = kubeclient.Get(ctx).CoreV1(). ...
```

you can see various example usages in `./pkg/auth/serviceaccount_test.go`